src/aura/STG_CMP_Tabs/STG_CMP_TabsHelper.js

```diff
@@ -193,6 +199,24 @@
 			}			
 
 		});
+	},
+
+	getOtherDisplay : function(component, value)
+	{
+		var prefix = component.get("v.namespacePrefix");
+		var label = '';
+		if(prefix === 'hed__')
+        {
```

Comment:
trivial issue. The indentation looks weird here

Reply:
hmm let me fix this.